<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminExtrasPopup</class>
 <widget class="QWidget" name="AdminExtrasPopup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>757</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QTableWidget {
	background: transparent;
	font: helvetica; 
	font-weight: bold;
}

QHeaderView::section
{
	spacing: 10px;
	background-color: gray;
	color: white;
	border: 1px solid ;
	margin: 1px;
	text-align: right;
	font-family: arial;
	font-size:12px;
}

QLineEdit {
	background: lightgray;
	border-radius: 10px;
	font: helvetica;
	font-weight: bold;
}

QPushButton {
	background: lightgray;
	border-radius: 10px;
	font-weight: bold;
}</string>
  </property>
  <widget class="QPushButton" name="pushButton_back">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QPushButton" name="MST">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Minimum Spanning Tree</string>
   </property>
  </widget>
  <widget class="QPushButton" name="DFS">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>90</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>DFS</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BFS">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>90</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>BFS</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>891</width>
     <height>601</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>90</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_distance">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>40</y>
     <width>201</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Distance:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
